taegis-xdr/operations.py

```diff
@@ -0,0 +1,231 @@
uests user_baseline total_spns user_avg_requests percentage_accessed hostname spns_accessed user_max_requests user source_address }} geo_ip {{ radius country_code_iso asn geohash ip_address longitude latitude }} watchlist_matches {{ details {{ reason attacks list_name }} entity }} login_failure {{ host target_address failed_auth_event user successful_auth_event source_address }} rare_program_rare_ip {{ host connections {{ source_ip destination_ip }} programs }} password_spray_detail {{ num_auth_failures num_auth_successes all_affected_users {{ target_user_name target_domain_name user_had_auth_success }} source_address }} hands_on_keyboard_details {{ host_id num_admin_events total_num_events matched_num_events common_parent_image_path matched_process {{ process_resource_id score event_time_sec image {{ image_path matched_features }} num_matched_features commandline {{ matched_features commandline }} severity }} username }} generic {{ generic {{ record {{ key value }} }} name }} dns_exfil {{ num_queries }} tactic_graph_detail {{ graph_id events {{ key values }} }} brute_force_detail {{ most_recent_auths_failures {{ resource_record_identifier action win_event_id domain event_timestamp target_username }} num_auth_successes last_successful_auth {{ resource_record_identifier action win_event_id domain event_timestamp target_username }} num_auth_failures }} ddos_source_ip {{ host_id sensor_id top_destination_ips {{ count ip_address }} analytic_observable_min_count event_observable_count historical_ip_counts {{ date {{ seconds nanos }} count }} event_observable_count_std_dev baseline_observable_count_std_dev baseline_observable_count_mean analytic_time_threshold analytic_observable_std_dev_threshold hour_partition baseline_num_days baseline_observable_count_median }} business_email_compromise {{ source_address_geo_summary {{ city {{ confidence name locale_names {{ record {{ key value }} }} geoname_id }} location {{ radius metro_code timezone longitude us_metro_code gmt_offset latitude }} asn {{ autonomous_system_org autonomous_system_no }} country {{ confidence code iso_code geoname_id }} continent {{ code geoname_id }} }} user_name source_address }} }} priority {{ version model_version model_name applied_time {{ seconds nanos }} value prioritizer evidence }} sensor_types events_metadata {{ began_at {{ seconds nanos }} first_event_id last_event_id updated_at {{ seconds nanos }} total_events ended_at {{ seconds nanos }} }} reference_details {{ reference {{ description url type }} }} group_key entities {{ relationships {{ to_entity relationship from_entity type }} entities }} event_ids {{ id }} tags suppressed resolution_history {{ user_id timestamp {{ seconds nanos }} id num_alerts_affected reason status }} tenant_id key_entities {{ entity label }} observation_ids {{ id }} visibility suppression_rules {{ id version }} alerting_rules {{ id version }} collection_ids {{ id }} status attack_technique_ids investigation_ids {{ id GenesisAlertsFlag }} }} part }}
```

Comment:
Can you put all the static code in a constant file

Reply:
Done